event/src/main/java/com/facebook/airlift/event/client/EventDataType.java

```diff
@@ -21,14 +21,16 @@
 import com.google.common.primitives.Primitives;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
-import org.joda.time.format.DateTimeFormatter;
```

Comment:
let us not remove any joda classes in this PR since we need backwards compatibility.

Reply:
I had to remove it and use the class directly because of name conflict with java.time.format.DateTimeFormatter on line 34. Please check line 187.

Reply:
For conflicting names you need to use the fully qualified name locally.